ci/docker/python-wheel-windows-test-vs2019.dockerfile

```diff
@@ -20,33 +20,8 @@
 
 # based on mcr.microsoft.com/windows/servercore:ltsc2019
 # contains choco and vs2019 preinstalled
-FROM abrarov/msvc-2019:2.11.0
-
-# hadolint shell=cmd.exe
```

Comment:
could you re-add this in order to try and fix the linting failures? This seems required as to what I can see here: https://github.com/apache/arrow/pull/43034/files

Reply:
Done!